ui-modules/app-inspector/app/components/entity-tree/entity-node.html

```diff
@@ -22,9 +22,12 @@
             <brooklyn-status-icon value="{{entity.serviceState}}" ng-if="entity.serviceState || entity.applicationId"></brooklyn-status-icon>
             <i class="fa fa-2x fa-external-link" ng-if="!entity.serviceState && !entity.applicationId"></i>
             <span>{{entity.name}}</span>
+            <span class="node-icon"><img ng-src="{{ iconUrl }}"/></span>
         </a>
-        <div class="entity-node-toggle" ng-if="entity.children.length > 0 || entity.members.length > 0" ng-click="onToggle($event)" >
+        <div class="entity-node-toggle-wrapper">
```

Comment:
Why wrapping this into another `div`? This entire "row" (i.e. entity status, entity name, icon and expand toggle) can be handled with flexbox which would be cleaner and easier to implement, rather than doing it with unnecessary HTML layout.

Reply:
the `ng-if` hides the toggle on leaf nodes.  but we want the icons to align so we need a placeholder/wrapper.